e2e/tests/global.setup.ts

```diff
@@ -1,102 +1,98 @@
-import { expect, test, test as setup } from '@playwright/test';
+import { expect, test } from '../fixtures/test-repo-fixture';
 import { LeftBarItems } from '../enums/left-bar-items.enum';
 import { VSCode } from '../pages/vscode.pages';
 import { COOLSTORE_REPO_URL, SCREENSHOTS_FOLDER } from '../utilities/consts';
 
-setup.describe(
-  'install extension and configure provider settings',
-  async () => {
-    let vscodeApp: VSCode;
+test.describe('install extension and configure provider settings', async () => {
```

Comment:
According to the [docs ](https://playwright.dev/docs/test-global-setup-teardown#setup) this should be `setup`